wasm-tools/pom.xml

```diff
@@ -47,6 +47,29 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>com.dylibso.chicory</groupId>
+        <artifactId>aot-maven-plugin-experimental</artifactId>
+        <executions>
+          <execution>
+            <id>wasmtools</id>
+            <goals>
+              <goal>wasm-aot-gen</goal>
+            </goals>
+            <configuration>
+              <name>com.dylibso.chicory.tools.wasm.WasmTools</name>
+              <wasmFile>${project.basedir}/target/wasm-tools.wasm</wasmFile>
+              <interpretedFunctions>
```

Comment:
Follow up task to document properly, and eventually improve, how to configure the Maven Plugin with the `interpretedFunctions` option.